tools/scrape/github.py

```diff
@@ -216,7 +216,7 @@ def scrape_plugin_repos(num):
     query = r.table('plugin_github_repos').filter({'is_blacklisted': False})
 
     # We don't want to scrape forks that not many people use.
-    query = query.filter(r.not_((r.row['is_fork'] == True) & (
+    query = query.filter(r.not_((r.row['is_fork'] is True) & (
```

Comment:
This possibility crossed my mind for half a second, and then I quickly dismissed it as paranoia. Thanks for catching it, and for the links. It's a pain that these things are so hard to test with automated tests. I wonder if we should just bite the bullet and have the DB running during tests.